docs/release-notes/8.6-beta.md

```diff
@@ -203,6 +203,12 @@ others:
 - subscription-manager
 - thunderbird
 
+## Removed packages
+
+The following packages were removed:
+
+- rhc and related packages 
+- redhat-insight and related packages
```

Comment:
`redhat-insight` or `redhat-access-insights`?

Reply:
@fixed you can merge